README.md

```diff
@@ -2,9 +2,23 @@
 
 ##### Feel free to send a PR to update this document.
 
-##### Requirements: (1) it is a company (for instance, you hire developers other than the company owners) and (2) it sponsors open source projects (e.g. through OpenCollective, Patreon, direct donations or time for employees).
-
-##### Each company should: (1) have a name, (2) have a link to blog post or page demonstrating that you sponsor some open source projects or employee hobby projects.
+<details>
+<summary><em>Click here to read the criteria to be on this list</em></summary>
+<p><strong>The goal of this list</strong> is to highlight companies which do the right thing by paying back to open source projects, because IT companies are often use hundreds of open source projects as a foundation for their software. Open source development is clearly very important for IT systems and products used world-wide, so sponsoring open source must become commonplace. This list is meant to encourage sponsoring practices.</p>
+  <h3>Definitions</h3>
+  <ul>
+    <li>COMPANY: a legal entity made to generate to profit.</li>
```

Comment:
The thing is I know a company who are non-profit and who does invest into open source and who tries to hire people and it would be unfair to them to not be able to get into that list.